src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/AdditionalApplicationPartAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Mvc.ApplicationParts
+{
+    [AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
+    public sealed class AdditionalApplicationPartAttribute : Attribute
```

Comment:
What would you think about renaming this? I don't really think `Additional` is a great term of art... What about `ProvideSattelitePartAttribute`

Reply:
`ProvideSupplementaryPartAttribute`? `Satellite` already has established semantic in .NET, I'd rather not use it here.

Reply:
RelatedAssemblyAttribute